src/components/svelte-web.ts

```diff
@@ -217,8 +238,8 @@ export default function registerWebComponent(
     }
 
     removeEventListener(event: string, callback: Callback) {
-      this.listeners.get(event)?.get(callback)?.()
-      this.listeners.get(event)?.delete(callback)
+      this.listenerRemovers.get(event)?.get(callback)?.()
+      this.listenerRemovers.get(event)?.delete(callback)
```

Comment:
Maybe a silly question, but should the `remover` function also be modified to always remove the callback from `listenerRemovers`?

Reply:
We could, I don't think it's critical though. On line 85 we don't need it to remove itself from the listeners array (though doing so wouldn't hurt) because we overwrite it in `addEventListener`. I could go either way on this though, wdyt is best